_tools/ic_people_website.md

```diff
@@ -0,0 +1,17 @@
+---
+layout: post
+title: IC People Website
+category: tools
+order: 2
+number_sections: TRUE
+---
+
+EPFL IC provides personal research page hosting under the `ic-people.epfl.ch/~name` domain.
+
+- Protocol: sftp
+- Host: ic-people-files.epfl.ch
+- Username: Gaspar Username
+- Password: Gaspar Password
+- Index folder: public_html
+
+Details about setup and templates can be found in IC's [github repository](https://github.com/EPFL-IC/ic-people-website).
```

Comment:
Suggest: “Details about setup, templates, and usage policies can be found in the official [IC-People GitHub repository](https://github.com/EPFL-IC/ic-people-website).”